websites/A/AnimeHeaven/presence.ts

```diff
@@ -0,0 +1,143 @@
+const presence = new Presence({
+    clientId: "816042675626442783"
+  }),
+  strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused"
+  }),
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+type VideoData = {
+  paused: boolean;
+  currentTime: number;
+  duration: number;
+};
+let iFrameVideo: VideoData;
+presence.on("iFrameData", (videoData: VideoData) => {
+  iFrameVideo = videoData;
+});
+
+const paths = {
+  async "/watch"(presenceData: PresenceData, buttons: boolean) {
+    const title = document.querySelector(".now2 .c a")?.textContent,
+      episode = document
+        .querySelector(".now2 .c")
+        ?.lastChild?.textContent?.match(/\s*-\s*(.+)/)[1];
+
+    presenceData.details = title || "Unknown title";
+    presenceData.state = episode || "Unknown episode";
+
+    if (buttons) {
+      const link = (document.querySelector(
+        ".now2 > div > a"
+      ) as HTMLLinkElement)?.href;
```

Comment:
instead of ``HTMLLinkElement`` use ``HTMLAnchorElement`` and delete ``?``

Reply:
Why delete `?`? It's a null check

Reply:
I don't think you will get it null + ``?`` would be like if .href wouldn't exist on this type, but it exists.